modulemd/include/modulemd-2.0/modulemd-module-index.h

```diff
@@ -246,6 +246,41 @@ modulemd_module_index_update_from_custom (ModulemdModuleIndex *self,
                                           GError **error);
 
 
+/**
+ * modulemd_module_index_update_from_defaults_directory:
+ * @self: This #ModulemdModuleIndex object.
+ * @path: (in): The path to a directory containing defaults documents.
+ * @strict: (in): Whether the parser should return failure if it encounters an
+ * unknown mapping key or if it should ignore it.
+ * @overrides_path: (in) (nullable): If non-NULL, the path to a directory
+ * containing defaults documents that should override those in @path.
+ *
```

Comment:
Sorry for not noticing this until now, but parameter @error isn't listed above.

Reply:
Thanks, fixed.